annotator-models/bin/run

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+#
+# A script to train the kaggle model remotely using ml-engine.
+#
+# To run with default hyperparameters from the kaggle-classification directory just enter:
+# './bin/run'
+#
+# To run with hyperparameter tuning, enter:
+# './bin/run -c hparam_config.yaml'
+#
+#
+# Setup Steps:
+# 1. Install the gcloud SDK
+# 2. Authenticate with the GCP project you want to use, `gcloud config set project [my-project]`
+# 3. Put the train and test data in Cloud Storage, `gsutil cp [DATA_FILE] gs://[BUCKET_NAME]/`
+#
+
+# Edit these!
+BUCKET_NAME=annotator_models
+CONFIG=cpu_config.yaml
+JOB_NAME=${USER}_kaggle_training
+
+# Note: this must be compatible with cells that have GPUs. us-central1 works.
+# See: https://cloud.google.com/ml-engine/docs/using-gpus
+REGION=us-central1
+DATE=`date '+%Y%m%d_%H%M%S'`
+OUTPUT_PATH=gs://${BUCKET_NAME}/models/${USER}/${DATE}
```

Comment:
this isn't consistent with the other run scripts, we should probably pull this common logic out (but you could just make it consistent for now)